### PR TITLE
Fix wheel creation via github action

### PR DESCRIPTION
**Pull request by uhurusurfa**

### CODE REVIEW

File: CONTRIBUTING.rst
Comment (Benbb96) [inline CONTRIBUTING.rst:106]: We can also add that it will be reviewed by one of the maintainer before merging

File: CONTRIBUTING.rst
Comment (uhurusurfa) [reply CONTRIBUTING.rst]: Fixed

File: CONTRIBUTING.rst
Comment (uhurusurfa) [reply CONTRIBUTING.rst]: Added

**Review by Benbb96 — APPROVED** [review]

**Pull request merged**
